tests/console/yast2_proxy.pm

```diff
@@ -30,42 +62,41 @@ sub run {
     assert_screen 'yast2_proxy_squid';
 
     # enable service start
-    send_key 'alt-b';
+    send_key_until_needlematch 'yast2_proxy_service_start', 'alt-b';    #Start service when booting
 
     # if firewall is enabled, then send_key alt-p, else move to page http ports
-    if (check_screen 'yast2_proxy_firewall_enabled') { send_key 'alt-p'; }
-    send_key 'alt-d';
+    if (check_screen 'yast2_proxy_firewall_enabled') { send_key 'alt-p', 1; }
+    wait_still_screen 3;
 
     # check network interfaces with open port in firewall
-    assert_screen 'yast2_proxy_network_interfaces';
-    send_key 'alt-n';
-    send_key 'alt-a';
-    send_key 'alt-o';
+    # repeat action as sometimes keys are not triggering action on leap if workers are slow
+    send_key_until_needlematch 'yast2_proxy_network_interfaces', 'alt-d', 2, 5;
+
+    wait_still_screen 1;
+    send_key 'alt-n', 1;
```

Comment:
We should the simply remove it from api and do replacement in the code. Or add this to documentation at least.